share/messages_to_error_log.txt

```diff
@@ -12352,6 +12352,23 @@ ER_GRP_RPL_MYSQL_NETWORK_PROVIDER_SERVER_ERROR_COMMAND_ERR
 # End of 8.1 error messages intended to be written to the server error log.
 #
 
+#
+# Start of Percona Server 8.0 server error log messages
+#
+start-error-number 48000
```

Comment:
did you check with @inikep about the error message number used by PS?

Reply:
I can check, but this has not changed. just the physical location in the file.